src/tutorials/lua_basics_lightbulb.md

```diff
@@ -770,205 +624,162 @@ There are two ways to change a trigger's label, by going to the trigger's **Prop
 
 ### Change the Interact Label to "Turn off Light"
 
```

Comment:
Add a step that changes it in Custom Properties first, to get a clear idea of what the interactionLabel actually is, and where it shows up.